server/app/routes/users.js

```diff
@@ -145,7 +145,7 @@ router.put('/:userId/addFriend', ensureAuthenticated, function(req, res, next) {
 
 router.put('/:userId/addPost', ensureAuthenticated, function(req, res, next) {
     User.findOne({ _id: req.params.userId })
-    .then(function(user){
+    .then(function(){
         return User.findOne({ _id: req.body.friend._id})
```

Comment:
i removed user from the parameter cause it was a lint error. not sure whats happening here either...